site/index.html

```
@@ -410,12 +474,125 @@ <h2 class="section-title">Go SDK</h2>
+                    <div class="code-window-header">
+                        <div class="code-window-dots">
+                            <span></span>
+                            <span></span>
+                            <span></span>
+                        </div>
+                        <span class="code-window-title">main.rs</span>
+                    </div>
+                    <pre class="code-window-content"><code><span class="code-keyword">use</span> drive9::Client;
+
+#[tokio::main]
+<span class="code-keyword">async fn</span> <span class="code-function">main</span>() -> Result&lt;(), drive9::Drive9Error&gt; {
```

HTMLHint (1.9.2)

[error] Special characters must be escaped : [ > ].

(spec-char-escape)